targeteditor/src/main/scala/explore/targeteditor/TargetQueries.scala

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.target
+
+import cats.effect.IO
+import cats.implicits._
+import clue.GraphQLQuery
+import crystal.react._
+import explore.implicits._
+import explore.model.SiderealTarget
+import explore.undo.Undoer
+import gem.Observation
+import gsp.math.Coordinates
+import gsp.math.Declination
+import gsp.math.Epoch
+import gsp.math.ProperMotion
+import gsp.math.RightAscension
+import io.circe.Decoder
+import io.circe.Encoder
+import io.circe.HCursor
+import io.circe.JsonObject
+import io.circe.generic.semiauto.deriveDecoder
+import io.circe.generic.semiauto.deriveEncoder
+import monocle.macros.Lenses
+
+object TargetQueries {
```

Comment:
Nice modularization! I like that all this is in a single place outside of the components.

Reply:
It made sense when I split the part reading/writing across components